Cargo.toml

```diff
@@ -29,7 +30,6 @@ optional = true
 [features]
 cm7 = []
 cm7-r0p1 = ["cm7"]
-inline-asm = []
```

Comment:
removing this is a breaking change, is the intent to release this as 0.8?

Perhaps it's better to leave it in with a note "deprecated, does nothing, kept for backwards compat"?

Reply:
Yes, from top of thread: "For cortex-m, I propose putting these changes in the upcoming 0.8 release (which is technically what the master branch is already on) and not backporting."

Reply:
The `inline-asm` features in the other crates could be removed then?

Reply:
> For cortex-m-semihosting/panic-semihosting I think a patch release would be ideal, especially since we had to yank the last c-m-sh release due to conflicting prebuilt binaries (a problem that should now vanish).

But I'm sort of on the fence there, we could do remove the feature and do a breaking release. For cortex-m-semihosting we already released and yanked 0.4 and are bank on 0.3, so we could drop the feature and do 0.4 there. Perhaps that's nicest.

Reply:
Pushed an update removing it from those crates too, may as well make a clean sweep of it.